src/vxfw/ScrollView.zig

```diff
@@ -0,0 +1,825 @@
+const std = @import("std");
+const vaxis = @import("../main.zig");
+
+const assert = std.debug.assert;
+
+const Allocator = std.mem.Allocator;
+
+const vxfw = @import("vxfw.zig");
+
+const ScrollView = @This();
+
+pub const Builder = struct {
+    userdata: *const anyopaque,
+    buildFn: *const fn (*const anyopaque, idx: usize, cursor: usize) ?vxfw.Widget,
+
+    pub inline fn itemAtIdx(self: Builder, idx: usize, cursor: usize) ?vxfw.Widget {
+        return self.buildFn(self.userdata, idx, cursor);
+    }
+};
```

Comment:
I had to make `itemAtIdx` public to be able to count the total number of children. Alternatively, I could add a `totalNumberOfChildren` function or something similar, but this was what came to mind first, so I did it this way.

Tim, do you think we need to keep this non-public?